app/src/main/java/org/xwiki/android/sync/utils/SystemTools.kt

```diff
@@ -42,6 +43,12 @@ fun getAppVersionName(context: Context): String {
     }
 }
 
+fun checkNet(context: Context): Boolean {
```

Comment:
I am not sure, but it is possible that will be better to use extension syntax like `fun Context.checkNet()`